src/nc.c

```diff
@@ -28,7 +28,7 @@
 #include <nc_conf.h>
 #include <nc_signal.h>
 
-#define NC_CONF_PATH        "conf/nutcracker.yml"
+#define NC_CONF_PATH        "/etc/nutcracker/nutcracker.yml"
```

Comment:
This is backwards incompatible, only works on `*nix`-like systems, and is unrelated to new redis command support. Remove it.

Reply:
what about defining something based on OS info?